game/scripts/npc/abilities/slark_essence_shift.txt

```diff
@@ -33,7 +33,7 @@
       "03"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "duration"                                        "15 30 60 100 150 180"
+        "duration"                                        "20 40 60 80 120 160" // OAA
```

Comment:
My goal was to buff early game Slark. Values after lvl 4 barely matter because fights are too fast. Also another reason for nerfing duration at later levels is because its annoying when you kill slark and walk around with lost stats for 4 minutes. That was my line of thought.